install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -819,6 +819,11 @@ data:
   # https://github.com/cilium/hubble/blob/master/Documentation/metrics.md
   hubble-metrics: {{- range .Values.hubble.metrics.enabled }}
     {{.}}
+{{- end }}
```

Comment:
I think the CI errors are because we're missing a closing `{{- end}}` for the [`range`](https://helm.sh/docs/chart_template_guide/control_structures/#looping-with-the-range-action). This `{{- end}}` closes the `{{- range [...] }}`, not the `{{- if .Values.hubble.metrix.enabled }}`. You need to close both, then add your `if` and `range`, and close both again.